elements/app-shell.html

```diff
@@ -46,6 +48,7 @@
         --highlight-pink: #e91e63;
         --highlight-blue: #2196f3;
         --input-border-color: #596c7a;
+        --header-height-fullscreen: 44px;
```

Comment:
The header resizing is actually really jarring imo. Could we leave it the same size, please?